airgun/entities/subscription.py

```diff
@@ -139,6 +139,19 @@ def read_subscription_manifest_expiration_date_only(self):
             manage_view.close_button.click()
         return manifest_expiration_date
 
+    def is_subscription_manifest_header_message_display(self):
```

Comment:
```suggestion
    def manifest_header_message_displayed(self):
```
Propose a shorter method name like this, or something similar?
Non-blocking

Reply:
This function is being used/called during testing subscription expiration date scenario, I think it makes more sense to use well descriptive name instead of shorter. This will return True or False based of condition "is header message currently displaying on page".